azurelinuxagent/ga/memorycontroller.py

```diff
@@ -158,6 +158,39 @@ def get_memory_usage(self):
         # In v2, cache memory is reported in the 'file' counter
         return self._get_memory_stat_counter("anon"), self._get_memory_stat_counter("file")
 
+    def get_unit_properties(self):
+        properties = super(MemoryControllerV2, self).get_unit_properties()
+        properties.append("MemoryHigh")
+        return properties
+
+    def get_memory_pressure_events(self):
```

Comment:
Will the agent reset the quota if process experienced too much time under memory pressure? Or are we just collecting this for telem only right now

Reply:
Just curious, why not reset the quota if too much time under memory pressure? Is it because we don't know how much time under memory pressure is 'too much' time without collecting telemetry first?